client/app/profile/profile.js

```diff
@@ -93,10 +102,26 @@ angular.module('knapsack.profile', ["ui.router"])
         }
         $modalInstance.dismiss("submit");
       }
+
+    };
+
+  // $scope.loadUser();
+  // $scope.loadFriends();
+
+var AboutMeController = function($scope, userForm, Profile, $modalInstance) {
```

Comment:
The AboutMeController already exists in your code on line 79. You did not resolve a merge conflict properly